### ISSUE TITLE
DeprecationWarning: inspect.getargspec()

### ISSUE THREAD

**Issue by rixx** [SmileyChris/easy-thumbnails#458]
DeprecationWarning is raised when running with Python 3.6:

```
lib/python3.6/site-packages/easy_thumbnails/utils.py:56: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() or inspect.getfullargspec()
  args = inspect.getargspec(processor)[0]
```

**Comment by jrief** [comment]
would you please be so kind and create a pull request, which works down until Python 2.7

**Comment by schinckel** [comment]
I believe this is addressed by #479, which still needs some work to work with all pythons.
